docs/source/nbclassic.rst

```diff
@@ -147,6 +84,3 @@ Jupyter Notebook 7.
 **Porting Notebook 6 Extensions**: Work being done in parallel. 
 ou can find a helpful list of classical Notebook extensions and corresponding Jupyterlab extensions
 if available at the `Jupyterlab-contrib website <https://jupyterlab-contrib.github.io/migrate_from_classical.html>`_.
-
-
-.. _JEP #79: https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html
```

Comment:
Since this link has been removed, maybe the syntax in the first line  under `NbClassic Timeline` can be removed? Or the link can be updated to the previously referenced JEP79 link "`Jupyter Enhancement Proposal #79`"?
<img width="1290" alt="Screen Shot 2023-02-17 at 8 12 30 AM" src="https://user-images.githubusercontent.com/12378147/219706242-6158ec84-ae74-4f12-827a-550f92b8c282.png">

Reply:
I have added back the link to the JEP 79